src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from "react";
+import { Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+import { AnyVenue, VenueTemplate } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import "./AdminVenues.scss";
+
+interface AdminVenuesProps {
+  venues: WithId<AnyVenue>[];
+}
+
+export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
+  const partyVenues = useMemo(() => {
+    return venues?.filter((venue) => venue.template === VenueTemplate.partymap);
+  }, [venues]);
+
+  const hasVenues: boolean = !!partyVenues.length;
+
+  return (
+    <div className="admin-venue">
+      <div className="admin-venue__create-button">
+        <h3>Admin Dashboard</h3>
+        <Button as={Link} to="/admin_v2/venue/creation">
+          Create a new space
+        </Button>
+      </div>
+      {!hasVenues && (
+        <div className="admin-venue__empty-venues">
+          <h3>Welcome!</h3>
+          <h3>Create your first Sparkle space</h3>
+        </div>
+      )}
+      {hasVenues && (
+        <div className="admin-venue__cards">
+          {partyVenues.map((venue, index) => {
+            return (
+              <div key={index} className="card">
+                <div
+                  className="card__bg"
+                  style={{
+                    backgroundImage: `url(${venue.mapBackgroundImageUrl})`,
+                    backgroundSize: "cover",
+                  }}
+                ></div>
+                <div className="card__info">
+                  <div
+                    className="card__logo"
+                    style={{ backgroundImage: `url(${venue.host?.icon})` }}
```

Comment:
This will render as `url(undefined)` or similar when `venue.host?.icon` isn't defined. We should have a different/better option for this.

Reply:
This is a required field when creating a venue. It will fallback to undefined if someone removes the icon from the database directly. That being said I'm still going handle it but we need to think about the other fields as well.

Reply:
I guess that raises a bigger question about what we should 'allow'/how we should write our types/etc. Eg. is 'someone broke it in the DB' a situation that our types should account for? Currently, since we don't validate venue config/enforce it when edited in the DB, I would opt for the more cautious side of things (eg. where we assume the worst and handle it accordingly)

If we were to add something that validated and enforced any DB changes to ensure they matched our standards (regardless of where those edits are made from), then we would be able to trust that a venue is in a 'valid state', and could opt for more lenient typing. This would be my preferred option of how to handle things, but obviously that requires a bunch of work that is definitely OOS of this PR.

Reply:
(backlogged the above in https://github.com/sparkletown/internal-sparkle-issues/issues/432)